keras_retinanet/preprocessing/generator.py

```diff
@@ -149,11 +131,29 @@ def next(self):
         for image_index, image in enumerate(image_group):
             image_batch[image_index, :image.shape[0], :image.shape[1], :image.shape[2]] = image
 
+        return image_batch
+
+    def anchor_targets(
+        self,
+        image_shape,
+        boxes,
+        num_classes,
+        mask_shape=None,
+        negative_overlap=0.4,
+        positive_overlap=0.5,
+        **kwargs
+    ):
+        return anchor_targets(image_shape, boxes, num_classes, mask_shape, negative_overlap, positive_overlap)
```

Comment:
Might rename the free function `anchor_targets` to reflect that it is used for bounding box regression (with labels).

Reply:
Done (renamed to `anchor_targets_bbox`).